src/estimation-kdl/CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ target_link_libraries(${libraryname} idyntree-core idyntree-kdl idyntree-sensors
 get_property(IDYNTREE_TREE_INCLUDE_DIRS GLOBAL PROPERTY IDYNTREE_TREE_INCLUDE_DIRS)
 target_include_directories(${libraryname} BEFORE PRIVATE ${IDYNTREE_TREE_INCLUDE_DIRS})
 
+if((${CMAKE_CXX_COMPILER_ID} MATCHES "GNU") OR (${CMAKE_CXX_COMPILER_ID} MATCHES "Clang"))
+    target_compile_options(${libraryname} PRIVATE "-Wno-deprecated")
```

Comment:
We won't change to CMake 3.5 until we have the icub images with Debian Stretch, therefore not this release and I fear that also the next might be problematic...